src/Mod/Sketcher/App/PythonConverter.cpp

```diff
@@ -312,18 +323,27 @@ PythonConverter::SingleGeometry PythonConverter::process(const Part::Geometry* g
                  for (auto& pole : poles) {
                      stream << "App.Vector(" << pole.x << "," << pole.y << "),";

```

Comment:
```suggestion
                     stream << "App.Vector(" << pole.x << ", " << pole.y << "), ";

```

Reply:
The former is way more readable. The way this works is:
1. We run this command in C++, it generates the python script.
2. This script is inserted on to the python console.
3. That script is thus run and acts on the application.
4. The script can be copied, possibly edited, and run instead of using the GUI. Alternatively this can be stored in a macro.

This is how it looks without the space:
![image](https://github.com/user-attachments/assets/a47c7e9d-4f09-4b5c-922e-e661764d9fa0)